src/test/java/com/ksm/wstbackoffice/WstBackofficeApplicationTests.java

```diff
@@ -1,13 +1,35 @@
 package com.ksm.wstbackoffice;
 
+import com.ksm.wstbackoffice.controller.AddressControllerTest;
+import com.ksm.wstbackoffice.controller.CountryControllerTest;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.jdbc.SqlGroup;
 
 @SpringBootTest
+@SqlGroup({
+		@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/beforeTestRun.sql"),
+		@Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/afterTestRun.sql")
+})
+@ActiveProfiles("test")
```

Comment:
It is create tables in database before test -> @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts="classpath:sql/beforeTestRun.sql"),

It is remove tables in database after test -> @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, scripts="classpath:sql/afterTestRun.sql")

chaged @ActiveProfiles("test") to Run/Debug Configurations -> VM options: -Dspring.profiles.active=test

Reply:
Why did you change it? Why do you need -Dspring.profiles.active=test?

Reply:
I changed because used it property -Dspring.profiles.active=test on previos projects.
It is JVM system parameter for setting test profile name paremeter which will be activated during application start-up 
We can use -Dspring.profiles.active=test or @ActiveProfiles("test")
https://www.baeldung.com/spring-profiles